### DISCUSSION TITLE
Visualization: mark the ID of the grain

### DISCUSSION THREAD

**Discussion by Ming-is** [neperfepx/neper#441] [Q&A] [answered]
Can the serial number of grain cells on the tessellation model be marked when visualized with neper?

**Comment by rquey** [comment, marked as answer]
No, not really.

To add the numbers to the PNG output, a possibility would be to write a pov file containing the numbers at the desired locations (which you may get using `-statcell id,x,y,z`) and include the file using `-includepov`. If you just would like to know which grain is where, a possibility is to write the tess file at `geo` format, and open it under Gmsh.
